Final/About.html

```diff
@@ -1,59 +1,50 @@
+    siblings and our father, we moved to Canada to live with our stepmom. I went to high school starting from grade 9. One of my classes was a computer class which was a mandatory.
+    Back then I wasn't interested in knowing more about computers rather than just being able to type faster than my classmates. I remember the first day of my comuputer class, completely oblivious of what my classmates were doing. Some of them were on hotmail chatting with friends and I thought I can just go to the hotmail in google and start chatting (God knows who I would've been chatting with!).</p>
+
+    <p>when my math teacher voluntarily taught how to make  games  using computers after school, I have always used to wonder how they do all those animations, and build websites but never attended the class because I thought it is too hard for me to learn it. The other reason I lost hope in learning computer was the language barrier. I came to Canada with no English language skills and I remember always trying to avoid people so they don't ask me questions and I do not know what to say back to them.</p>
+
+    <p>I graduated from highschool in 2010 and started Universtiy in 2011. Since I was not interested in the program I was taking (Health administrative), I took a year off and started going to college for dental assisting level II program. The overall program was very interesting and I graduated with honor in 2014. I got married to my amazing husband in 2013 who is a U.S citizen. I have always being a person who wants to learn and know how things work, and learning how the computer function was one of them. I had my first child in October 2015 and the second child in September 2017. My husband sponsered me to U.S and now I am living in Atlanta, Georgia.</p>
+
+    <p>Since I became stay-home mother, my flexibility for learning and working has became too tight. My family member told me about Career Hub when I was looking for a part-time job. Fortunately, I found out about this amazing computer coding program at the Career Hub given by very amazing, helpful, and caring instructor and TAs. I was very excited and joined the team without hesitation. I have been having some obstacles attending the class but, I have promised not to give up and try my best as much as I can to complete and be proud of myself. Fortunately, everyone in this program has been very helpful which keeps my hopes up all the time. I have came to this so far, learned what html, css, and java scrip is and what they also do. I am still learning and knowing what it is like to be a web developer.</p>
   </section>
+
+  <div>
     <div>
-        <span><a href="https://github.com/UmmuAyyub" target="_blank" style="color:red">GitHub</a></span><br>
-        <span><a href="https://www.linkedin.com/in/nezira-usmael-6358b3182/" target="_blank" style="color:red">linkedin</a></span>
-        </div>
-    <section></section><br><br>
```

Comment:
Using `div` instead of `span` means you don't need `br` tags.

Reply:
Thank you Julie.